.travis.yml

```diff
@@ -9,8 +9,9 @@ dist: xenial
 language: rust
 
 install:
-  - nvm install 10 && nvm alias default 10 && npm install -g yarn@latest && yarn --version
-  - (cd dex-contracts && yarn --frozen-lockfile && yarn run prepack)
+  - nvm install 12 && nvm alias default 12 && npm install -g yarn@latest && yarn --version
+  # Yarn exits with non zero exit code because an optional dependency does not compile.
+  - (cd dex-contracts && yarn --frozen-lockfile && yarn run prepack || true)
```

Comment:
This seems pretty intense... What if the install actually fails?

Reply:
Then the `script` step which builds and tests our code is going to fail because it can't find the generated contract json artifacts.

Reply:
Excellent reasoning!

Reply:
Weird.. for me yarn exits with code 0.